### PR TITLE
Bugfix/e UI 3732 breathing space data not being persisted on previous button

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3732

### Change description ###
Breathing space is not being persisted when you press the previous button. To fix this will set `caseEventData.data = caseEventData.event_data` inside `toPreviousPage()` method this saves the state from the previous screens.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
 
   toPreviousPage() {
     let caseEventData: CaseEventData = this.buildCaseEventData();
+    caseEventData.data = caseEventData.event_data;
```
Comment (paul-graham) [inline src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts:121]: Can you put a comment in here explaining what this is for?

**Review by paul-graham — CHANGES_REQUESTED** [review]
I think the main change needs an explanatory comment as it's not clear why it's needed.

Also, this one can't be merged into `master` yet as it includes code from another PR that is still going through QA.

File: src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts (hunk 1 above)
Comment (kuda1992) [reply src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts:121]: ah yes

**Review by ishOsw — DISMISSED** [review]
looks good

**Review by paul-graham — COMMENTED** [review]
Thanks for making the changes, @kuda1992. All looks good.

I'm going to keep the "Request changes" status just for the moment, though, pending that other PR getting merged into `master` first, as discussed. :)

**Review by paul-graham — APPROVED** [review]
Needs a rebase but otherwise looks fine. Will re-review following the rebase.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.79.7",
+  "version": "2.79.6-breathing-data-not-being-persisted",
```
Comment (Daniel-Lam) [inline package.json:3]: @kuda1992 - minor point, but consider renaming the suffix as `breathing-space-data-not-being-persisted`, for consistency with the PR title.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@kuda1992 - just a minor comment on the suffix name

File: package.json (hunk 2 above)
Comment (kuda1992) [reply package.json:3]: @Daniel-Lam is this important or just a convention I should carry out going forward?

Comment (Daniel-Lam) [reply package.json]: @kuda1992 - not really that important, just that it looked a bit strange when it said "breathing data" 😄. Probably a good idea, nonetheless, to name the tag suffixes descriptively.

**Review by Daniel-Lam — DISMISSED** [review]

**Review by phillip-whitaker — APPROVED** [review]

**Review by rayliangatos — APPROVED** [review]

**Review by phillip-whitaker — APPROVED** [review]
looks normal

**Review by paul-graham — APPROVED** [review]
Pure update to the `RELEASE-NOTES.md` as the code is already in there.

**Pull request merged**